config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :companies
```

Comment:
root usually comes first in the routes. So this line would be moved down under devise. The routes is a priority list of the most important ones at the top and lower ones coming down.